samcli/lib/cookiecutter/template.py

```diff
@@ -121,6 +121,7 @@ def run_interactive_flows(self, context: Optional[Dict] = None) -> Dict:
         """
         try:
             context = context if context else {}
+            context["shared_values"] = "default"
```

Comment:
Can you explain why this is needed? And can you also confirm that it won't break `sam init` flow?

Reply:
No it won't break the `sam init` flow. This value is used in `sam pipeline init` templates to utilize previous answer to skip some questions.